azure/ConsiderationsForServiceDesign.md

```diff
@@ -234,7 +235,7 @@ PATCH must never be used for long-running operations -- it should be reserved fo
 If a long-running update is required it should be implemented with POST.
 
 There is a special form of long-running operation initiated with PUT that is described
-in [Create (PUT) with additional long-running processing](#create-put-with-additional-long-running-processing).
+in [Create (PUT) with additional long-running processing](./Guidelines.md#put-operation-with-additional-long-running-processing).
```

Comment:
In our SDK markdowns, we're not allowed to use relative references because they don't work in all contexts. Recommend a bookmarked aka link i.e., <https://aka.ms/azapi/guidelines#put-operation-with-additional-long-running-processing>.

Reply:
```suggestion
in [Create (PUT) with additional long-running processing](https://aka.ms/azapi/guidelines#put-operation-with-additional-long-running-processing).
```